packages/contracts/contracts/optimistic-ethereum/libraries/standards/L2StandardERC20.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.5.16 <0.8.0;
+
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+import './UniswapV2ERC20.sol';
+
+contract L2StandardERC20 is UniswapV2ERC20, Ownable {
+    address public l1Token;
+
+    /**
+     * @param _l1Token Address of the corresponding L1 token.
+     * @param _name ERC20 name.
+     * @param _symbol ERC20 symbol.
+     */
+    constructor(
```

Comment:
We have to initialize this contract for the `Ownable` anyway right?  I am concerned that these constructor args will cause unsafe code in some cases.  I think because of this it may actually be necessary to pass `0x0000` to the constructor for `UniswapV2ERC20` and then having all of the variables initialized with a separate transaction.

Reply:
Can we enforce any checks in the Gateway for this? There are the OZ ERC20 implementation constructor arguments.